MathlibTest/CountHeartbeatsLinter.lean

```diff
@@ -0,0 +1,18 @@
+import Mathlib.Util.CountHeartbeats
+
+-- sets the `countHeartbeats` linter option to `true`
+#count_heartbeats
```

Comment:
is it really useful to have this instead of simply writing `set_option linter.countHeartbeats true`? The latter seems more idiomatic

Reply:
I think that for discoverability, the alternative spelling `count_heartbeats` could be useful.  In the long run, it probably does not matter either way.  I'll poll mathlib, to see if there are strong views either way.

Reply:
Your reason to add the command makes sense, especially after I noticed that `count_heartbeats in` is valid already